jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -27,16 +27,17 @@
 from jdaviz.configs.cubeviz.plugins.viewers import WithSliceIndicator
 
 
-__all__ = ['SpectralExtraction']
+__all__ = ['CubeSpectralExtraction']
 
 
 @tray_registry(
-    'cubeviz-spectral-extraction', label="Spectral Extraction"
+    'cube-spectral-extraction', label="Cube Spectral Extraction"
 )
-class SpectralExtraction(PluginTemplateMixin, ApertureSubsetSelectMixin,
-                         DatasetSelectMixin, AddResultsMixin):
+class CubeSpectralExtraction(PluginTemplateMixin, ApertureSubsetSelectMixin,
+                             DatasetSelectMixin, AddResultsMixin):
```

Comment:
technically I don't think we need to rename the classes if anyone has concerns about breaking links in docs, etc.  But it might be worth it to do it just to keep ourselves sane.

Reply:
I know this has already been discussed, but my two cents is that the class name here might as well be `SpectralExtraction3D` to mirror the 2D case, since "cube" isn't in the internal registry label or API label.